recipes/graylog-forwarder/recipe.rb

```diff
@@ -0,0 +1,53 @@
+require_relative '../tools'
+
+class GraylogForwarder < FPM::Cookery::Recipe
+  include Tools
+
+  description 'Graylog Forwarder'
+
+  name     'graylog-forwarder'
+  version  data.version
+  revision data.revision
+  homepage data.homepage
+  arch     'all'
+
+  source data.source
+  sha256 data.sha256
+
+  maintainer data.maintainer
+  vendor     data.vendor
+  license    'see https://www.graylog.org/enterprise'
+
+  targets :deb do
+    section 'net'
+    config_files '/etc/graylog/forwarder/forwarder.conf', '/etc/graylog/forwarder/jvm.options'
+  end
+
+  targets :rpm do
+    config_files '/etc/graylog/forwarder/forwarder.conf', config_files '/etc/graylog/forwarder/jvm.options'
```

Comment:
There is a stray `config_files` statement. :wink: 
```suggestion
    config_files '/etc/graylog/forwarder/forwarder.conf', '/etc/graylog/forwarder/jvm.options'
```

Reply:
Ahhh! Can't believe I missed that.